software/Test_Robot_Software/main.py

```diff
@@ -0,0 +1,93 @@
+from camera import RealsenseCamera
+from image_processor import ImageProcessor
+from api import RobotMovement
+from robot_state import State
+from state_machine import StateMachine
+from time import time
+
+import cv2
+
+class Main:
```

Comment:
If you plan to use this as the main class, it would be nice to see the loop code inside here as well since they should logically be connected